tubesync/sync/tests.py

```diff
@@ -1719,68 +1719,28 @@ def test_delete_old_media(self):
 
 class TypeDirectoryPathTestCase(TestCase):
     def setUp(self):
-        self.audio_dir = Path(settings.DOWNLOAD_AUDIO_DIR)
-        self.video_dir = Path(settings.DOWNLOAD_VIDEO_DIR)
-        self.download_dir = Path(settings.DOWNLOAD_ROOT)
         self.source = Source(
             directory="test_directory",
             source_resolution=Source.SOURCE_RESOLUTION_AUDIO,
         )
 
-    def test_default_mode_audio(self):
+    def test_directory_prefix_default(self):
         """
-        Test default mode for audio resolution.
+        Test that default directory prefix exist.
         """
-        os.environ["TUBESYNC_DIRECTORY_MODE"] = "default"
-        expected_path = self.audio_dir / self.source.directory
-        self.assertEqual(self.source.type_directory_path, expected_path)
+        os.environ['TUBESYNC_DIRECTORY_PREFIX'] = ''
```

Comment:
Is this needed for the default case? In other words, we should already be in the default scenario just by loading settings.